Dockerfile

```diff
@@ -52,5 +52,35 @@ COPY --from=nim-build /app/migrations/ /app/migrations/
 RUN ln -sv /usr/local/bin/$MAKE_TARGET /usr/bin/wakunode
 
 ENTRYPOINT ["/usr/bin/wakunode"]
+
 # By default just show help if called without arguments
 CMD ["--help"]
+
+
+# DEBUG IMAGE ------------------------------------------------------------------
+
+# Build debug tools: heaptrack
+FROM alpine:edge AS heaptrack-build
+
+RUN apk update
+RUN apk add -- gdb git g++ make cmake zlib-dev boost-dev libunwind-dev
+RUN git clone https://github.com/KDE/heaptrack.git /heaptrack
+
+WORKDIR /heaptrack/build
+RUN cmake -DCMAKE_BUILD_TYPE=Release ..
+RUN make -j$(nproc)
+
+
+# Debug image
+FROM prod AS debug
+
+RUN apk add --no-cache gdb
+
+# Add heaptrack
+COPY --from=heaptrack-build /heaptrack/build/ /heaptrack/build/
+COPY --from=heaptrack-build /usr/lib/ /usr/lib/
```

Comment:
@Ivansete-status 

This is weird, I would not blindly copy `/usr/lib/` between images (especially different versions of images) as you might mess up some libraries.

I assume you are doing this because heaptrack actually drops some files in there? It would be good to know which files and only copy those rather than whole  `/usr/lib`

Reply:
I see the point. I will check it asap to have only the needed files copied.
Thanks @vpavlin !

Reply:
Disregard, there are some libs missing when you don't include /usr/lib :)